File: PrimeNumber.java
import java.util.*;
class PrimeNumber
{
    public static void main(String[]args)
    {
        int n,status=1,num=3;
        Scanner sc=new Scanner(System.in);
        System.out.println("Enter the number: ");
        n=sc.nextInt();
        if(n>=1){
            System.out.println("First "+n+" prime are: ");
            System.out.println();
        }
        for(int count=2;count<=n;){
            for(int i=2;i<=num-1;i++){
                if(num%i==0){
                    status=0;
                    break;
                }
            }
            if(status!=0){
                System.out.println(num);
                count++;
            }
            status=1;
            num++;
        }
    }
}

File: PrimeNumber.c
int main()
{
    int n,i,x=3,count;
    printf("Enter the number: \n");
    scanf("%d",&n);
    if(n>=1){
        printf("Enter %d the prime number are: \n",n);
        printf("2\n");
    }
    for(count=2;count<=n;){
        for(i=2;i<=x-1;i++){
            if(x%i==0)
                break;
        }
        if(i==x){
            printf("%d\n",x);
            count++;
        }
        x++;
    }
    return 0;
}
